From 8e089ae8249899b8e47cbe7abfe3dd8814458c89 Mon Sep 17 00:00:00 2001
From: Tim Writer <Tim.Writer@amd.com>
Date: Mon, 25 Nov 2013 15:51:21 -0500
Subject: [PATCH 08/11] configure: fix first run of autogen/automake

Without AC_CONFIG_AUX_DIR, libtoolize fails to copy ltmain.sh,
causing the first run of automake to fail.
---
 configure.ac |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/configure.ac b/configure.ac
index fcf4d9d..6dcca1c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -9,6 +9,10 @@ AC_INIT(GStreamer OpenMAX Plug-ins, 1.0.0.1,
     http://bugzilla.gnome.org/enter_bug.cgi?product=GStreamer,
     gst-omx)
 
+dnl Forces libtoolize to copy ltmain.sh. Without it, automake fails on
+dnl the first run.
+AC_CONFIG_AUX_DIR([.])
+
 AG_GST_INIT
 
 dnl initialize automake
-- 
1.7.9.5

